torch/testing/_internal/common_modules.py

```diff
@@ -984,6 +985,13 @@ def module_inputs_torch_nn_RNN_GRU(module_info, device, dtype, requires_grad, tr
                 reference_fn=partial(no_batch_dim_reference_rnn_gru, batch_first=b_f),
             )
         )
+        samples.append(
+            ModuleInput(
+                constructor_input=FunctionInput(**cons_args),
+                forward_input=FunctionInput(pack_padded_sequence(make_input((5, 2, 2)), torch.tensor([5, 3]))),
```

Comment:
nit: The code looks correct to me, but can you add something with a slightly larger batch size (than 2)? Maybe 5?